layouts/partials/footer.html

```diff
@@ -12,28 +12,32 @@
       <div class="col-6 col-sm-4 text-right text-xs-center order-sm-3">
         {{ with $links }}
         {{ with index . "developer"}}
-        {{ template "footer-links-block"  . }}
-        {{ end }}
+        <ul class="list-inline mb-0">
+          {{ template "footer-links-block"  . }}
+            <li class="list-inline-item mx-2" >&nbsp</li>
+            <li class="list-inline-item mx-2" >
+              <img src="/img/powered-gvisor.png" alt="Powered by gVisor">
+            </li>
+          {{ end }}
```

Comment:
I don't think the new Powered by gVisor should be inside the footer-links-block template. You should be able to move this line to right after the template line and then have the two new list items.

Reply:
If I move it out it puts the image wither below or above the footer-links-block which is not what I want. I actually don't care how it's done, as long as it's in the right location.

Reply:
Just tried my suggestion and the rendered site is exactly the same.

Here is the code I was suggesting in case it wasn't clear:
```
        <ul class="list-inline mb-0">
          {{ template "footer-links-block"  . }}
          {{ end }}
          <li class="list-inline-item mx-2" >&nbsp</li>
          <li class="list-inline-item mx-2" >
            <img src="/img/powered-gvisor.png" alt="Powered by gVisor">
          </li>
        </ul>
```